documentation/asciidoc/computers/raspberry-pi/spi-bus-on-raspberry-pi.adoc

```diff
@@ -346,7 +346,6 @@ Setup and hold times related to the automatic assertion and de-assertion of the
 * The CS line will be asserted at least three core clock cycles before the msb of the first byte of the transfer.
 * The CS line will be de-asserted no earlier than one core clock cycle after the trailing edge of the final clock pulse.
 
-[[software]]
```

Comment:
Wad this (seemingly unrelated) change intended to part of this PR?

Reply:
Sadly yes. I'm not sure when it happened, but this anchor started to throw a warning as a duplicate in the last couple of weeks. I'm sure it has something to do with something I've recently changed, but it's a _really_ vague anchor point for a very specific subject, so I figure we're better off letting this section use the automatic anchor instead 😉